js/apps/filter-page/directives/filter.js

```diff
@@ -11,7 +11,7 @@
 			if (inputType === 'radio') {
 				$input.prop('name', $scope.filterFamily.name);
 			}
-
+						
```

Comment:
Intentional whitespace?

Reply:
@ascire yes, this probably isn't the greatest layout, but i think we prefer to keep white space between conditions. also between methods, unless they are one liners...something we should probably iron out though.

Reply:
The thing that struck me was it appears to have changed from an empty line to 6 tabs.  You know there are [languages](https;//www.python.org) where this [makes a difference](https://en.wikipedia.org/wiki/Whitespace_(programming_language)), right?

Reply:
I see, yea thats not right.